catroidTest/src/org/catrobat/catroid/uitest/content/brick/AddItemToUserListTest.java

```diff
@@ -91,7 +92,7 @@ public void testCreateNewUserListAndDeletion() {
 		assertTrue("ScriptFragment not visible", solo.waitForText(solo.getString(R.string.brick_add_item_to_userlist)));
 		assertTrue("Created List not set in spinner", solo.searchText(userListName));
 
-		UserList userList = (UserList) Reflection.getPrivateField(addItemToUserListBrick, "userList");
+		UserList userList = (UserList) Reflection.getPrivateField(UserListBrick.class, addItemToUserListBrick, "userList");
```

Comment:
It would also be possible to get private fields which are declared in a parent class without specifying the class. But I didn't implement it back then to protect ourselves. Maybe a little overcautious? Should I add this feature?